app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>A user's profile</h1>
\ No newline at end of file
+<div class="user-item">
+  <img src=<%= @user.photo %> alt="profile-image">
+  <div class="user-info">
+    <h2><%= @user.name%></h2>
+    <p>Number of Posts: <%= @user.posts_counter || 0 %></p>
+  </div>
+</div>
+<div class="bio">
+  <h3>Bio</h3>
+  <p><%= @user.bio %></p>
+</div>
+
```

Comment:
There should be `See all Post` link that redirect the user to the user's post index.